### PR TITLE
Documentation: Add VSCode debugger instructions

### PR DESCRIPTION
**Pull request by kostyafarber**

### CODE REVIEW

File: Documentation/EditorConfiguration/VSCodeConfiguration.md [hunk 1]
```diff
@@ -286,6 +286,32 @@ The following three example tasks should suffice in most situations, and allow y
...
+}
+```
+
+then run Ladybird with the debug preset and with the `--debug-process WebContent` flag:
+
+```bash
+CC=$(brew --prefix llvm)/bin/clang CXX=$(brew --prefix llvm)/bin/clang++ BUILD_PRESET=Debug ./Meta/ladybird.sh run ladybird --debug-process WebContent
+```
+
+running Ladybird in this way will pause execution until a debugger is attached which you can run if you go to the Run and Debug menu and run the Attach to WebContent configuration.
```
Comment (sideshowbarker) [inline Documentation/EditorConfiguration/VSCodeConfiguration.md:313]:
```suggestion
Running Ladybird in this way will pause execution until a debugger is attached. You can then run the debugger by going to the **Run and Debug** menu and selecting the **Attach to WebContent** configuration. 
```
(just a non-maintainer review suggestion)

**Comment by kostyafarber** [comment]
@sideshowbarker nice suggestion. I think it would be nice to add a snippet about Linux (essentially the path to WebContent is different). Would need someone that uses Linux and VSCode to confirm, I can ask the Discord.

**Comment by kostyafarber** [comment]
Linux instructions added.

**Review by ADKaster — APPROVED** [review]
Thanks! Hopefully we can flesh out the debug info so that everyone can benefit. I know personally, I'm a command line gdb person :)

**Pull request merged**